Source/main.py

```diff
@@ -0,0 +1,53 @@
+import os
+import subprocess  # noqa
+
+import getch
+from clean_user_data import cleaning
+from get_prediction import create_plots
+
+
+headline = """
+############################################################
+#                    Sentiment analysis                    #
+#                       with tweets                        #
+############################################################ """  # noqa
+
+
+def get_twits(name):
+    os.mkdir(f"../Users/{name}")
+    comand = f"twint -u {name} --since 2020-01-01 --until 2020-12-31 -o ../Users/{name}/{name}.csv --csv"
```

Comment:
Hardcoded year does not look good. Add it as a keyword argument with default value.